client/src/components/Context.js

```diff
@@ -115,6 +117,15 @@ function getRoutes(user) {
       icon: IconPermIdentity,
       disabled: false,
     },
+
+    {
+      name: 'Capture Matching',
+      linkTo: '/captures',
+      component: CaptureMachingFrame,
+      icon: CompareIcon,
+
+      disabled: false,
```

Comment:
Later on, we may need to create a new user policy for capture matching, but for now I'd suggest restricting this to super users and tree approvers:
```js
      disabled: !hasPermission(
        user, [
          POLICIES.SUPER_PERMISSION,
          POLICIES.APPROVE_TREE,
        ]),
```